src/Documents/Changes/DatabaseChanges.ts

```diff
@@ -453,13 +459,20 @@ export class DatabaseChanges implements IDatabaseChanges {
                 break;
             case "TopologyChange":
                 const topologyChange = value as TopologyChange;
-                if (!this._requestExecutor) {
-                    const serverNode = new ServerNode();
-                    serverNode.url = topologyChange.url;
-                    serverNode.database = topologyChange.database;
+                const requestExecutor = this._requestExecutor;
+                if (requestExecutor) {
+                    const node = new ServerNode({
+                        url: topologyChange.url,
```

Comment:
matching c# version: https://github.com/ravendb/ravendb/blob/v5.0/src/Raven.Client/Documents/Changes/DatabaseChanges.cs#L714

I'd like to avoid case when new prop comes to server node, etc.